src/context/FeedManager.tsx

```diff
@@ -146,6 +146,10 @@ export class FeedManager<
     this.state.lastReverseResponse = { next: previousUrl };
   }
 
+  setOptions(newOptions: Partial<GetFeedOptions>) {
+    this.props.options = this.getOptions(newOptions);
```

Comment:
last one, cant't we just do?
```suggestion
    this.props.options = newOptions;
```

Reply:
Actually... true, I was overthinking that way too much. I'm going to get rid of the method as well, we can set it from the `Feed`.